ci/gpu/build.sh

```diff
@@ -31,7 +32,8 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install -c nvidia/label/cuda$CUDA_VERSION_SHORT -c rapidsai/label/cuda$CUDA_VERSION_SHORT -c rapidsai-nightly/label/cuda$CUDA_VERSION_SHORT -c numba -c conda-forge -c defaults cudf=$CUDF_VERSION nvgraph networkx python-louvain
+conda install -c nvidia/label/cuda$CUDA_REL -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c numba -c conda-forge \
+    cudf=$CUDF_VERSION rmm=$RMM_VERSION nvgraph networkx python-louvain
```

Comment:
Notice that for 0.7 release nvgraph package needs to be updated as discussed on slack and in #194